entity/templates/src/main/java/package/domain/_Entity.java

```diff
@@ -171,7 +170,7 @@
     <%_ } else { _%>
     <%_     if (relationships[relationshipId].ownerSide) { _%>
     @OneToOne
-    @JoinColumn(name = "<%= getColumnName(relationships[relationshipId].relationshipName) %>_id", unique = true)
```

Comment:
Yes,  in fact the PR is to make liquibase use the same naming strategy that we configure for spring in application.yml